<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggingSettingsDialog</class>
 <widget class="QDialog" name="LoggingSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="logSettingsAppplyButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="loggingEnabledCheckbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>enabled</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>File Path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fileLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="selectFile">
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
